Editor/UI/EditorWindows/SettingsWindow/SettingsEditor.cs

```diff
@@ -52,7 +52,7 @@ public static void ShowWindow()
         {
             var window = GetWindow<SettingsEditor>();
             window.titleContent = new GUIContent(SETTINGS);
-            window.minSize = new Vector2(500, 580);
+            window.minSize = new Vector2(500, 620);
```

Comment:
I don't love these inline, hard coded size values

Reply:
Any suggestions on how to restrict min size for window?